py_zipkin/zipkin.py

```diff
@@ -113,7 +113,8 @@ def __init__(
         add_logging_annotation=False,
         report_root_timestamp=False,
         use_128bit_trace_id=False,
-        host=None
+        host=None,
+        firehose_handler=None
```

Comment:
Do we want people to re-implement the firehose handler then? I thought we had decided to implement it in py_zipkin directly. In that case this could just be a boolean flag.

Reply:
Personally, I think we should keep it as a reference to a function. I'd rather see a default implementation that people can import and pass as an argument. That way, users can override the default behaviour.

Reply:
I will add a reference implementation here once we have a design doc and spec for the local firehose daemon. But, at least for now, I agree with @msindwan that we can leave it up to the user.